source/includes/code/keyrotate.yaml

```diff
@@ -0,0 +1,35 @@
+keyrotate:
+  apiVersion: v1
+  bucket: bucket
+  prefix: 
+  encryption:
+    type: sse-kms # valid values are sse-s3 and sse-kms
+    
+    # following encryption values only valid for sse-kms type
```

Comment:
Should we note that sse-s3 uses the deployment key set using...whatever the envvar is for that?

Reply:
Also do we ignore or exit-error if you define `type: sse-s3` and also set the following?

Reply:
Added a note for the first comment.
Will have to ask/test to figure out the second.

Reply:
> Also do we ignore or exit-error if you define `type: sse-s3` and also set the following?

@harshavardhana What's the result here to Ravind's question? In the` sse-s3` type, do we ignore any values that might be entered in `key` and `context`? Or does it cause an error?

Reply:
The code doesn't have any verification on these fields when set to `sse-s3`, so `key` and `context` are just ignored if set. (@harshavardhana please correct me if I'm wrong.)

Updating the code sample with an extra comment line to that effect.